.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2.1
 executors:
   js-exec:
     docker:
-      - image: circleci/node:12.22
+      - image: cimg/node:14.19
```

Comment:
Consider bumping this (and the others) to node 16 as it's the active LTS branch.

Reply:
Good point, might as well now that I'm bumping it anyway :)